### ISSUE TITLE
Web component: Masthead-With custom data - On iPhone voice over,  all the options which has sub options, focus is not set to the first interactive element

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#5147]
## Detailed description

Issue:
1. After expanding the focus is not set to the   first interactive element, for screen reader users the focus stays on the triggering element  which is not visually present.
2. When Level 1 menu options are displayed, screen reader also identifies the content from the level 0 which is visually not present. this issue happens in all the menu levels as well.

**Comment by annawen1** [comment]
@Praveen-sr Can this issue be tested again? I believe the two bugs in this ticket have been resolved.

**Comment by Praveen-sr** [comment]
@annawen1 Re tested this on Web component,  Vo iPhone-original issue fixed. But now we see a regression issue here, 

Issue: When the Hamburger is open, the parent page elements are identified. e.g. skip to main content link, IBM logo, Search button and User profile.
Recommendation: When the Hamburger is open, the parent page elements should be hidden from the screen reader. 

cc @RobertaJHahn

**Comment by RobertaJHahn** [comment]
The new issue mentioned by Praveen above is addressed in issue #4776.  This issue can be closed.
